README.md

```diff
@@ -278,7 +317,7 @@ workflow test {
 Generate a template `hello.json` file with the `inputs` subcommand:
 
 ```
-$ java -jar target/scala-2.11/cromwell-0.7.jar inputs hello.wdl
+$ java -jar target/scala-2.11/cromwell.jar inputs hello.wdl
```

Comment:
Or at least, make the examples start from the same working directory (above it was ../target/....)